app.js

```diff
@@ -0,0 +1,62 @@
+const express = require("express");
+const Sequelize = require("sequelize");
+var app = express();
+
+const bodyParser = require("body-parser");
+
+const urlencodedParser = bodyParser.urlencoded({extended: false});
+
+const sequelize = new Sequelize("postgres", "postgres", "1234567", {
+  dialect: "postgres"
+     });
+ 
+// определяем модель User
+const User = sequelize.define("user", {
```

Comment:
Move models to a separate models folder and separate files. Use sequelize cli to create index.js file for models.
Remove comments from the code